frontend/public/components/utils/headings.tsx

```diff
@@ -132,7 +132,7 @@ export const ConnectedPageHeading = connectToModel(
     ) : (
       <>
         {hasButtonActions && hasData && (
-          <ActionButtons actionButtons={buttonActions.map((a) => a(kindObj, data))} />
+          <ActionButtons actionButtons={buttonActions.map((a) => a())} />
```

Comment:
ConnectedPageHeading is used in places other than the secrets page. do we need this change for the PR?